lcf2xml/src/main.cpp

```diff
@@ -42,38 +55,44 @@ int main(int argc, char** argv)
 {
 	if (argc <= 1)
 	{
-		std::cerr << "Lcf2Xml - Converts RPG Maker 2k/2k3 Files to XML and vice versa" << std::endl;
-		std::cerr << "Usage: lcf2xml [FILE] [OUTFILE]" << std::endl;
-		//std::cerr << "If OUTFILE is missing the resulting file is named FILE with the extension replaced" << std::endl;
+		std::cerr << "LCF2XML - Converts RPG Maker 2000/2003 Files into XML and vice versa" << std::endl;
+		std::cerr << "Usage: " << argv[0] << " filename" << std::endl;
+
 		return 1;
 	}
 
-	std::string infile;
+	std::vector<std::string> infiles;
 	std::string outfile;
 
-	infile = argv[1];
-
+	FileCategories category = FileCategory_Invalid;
 	FileTypes type;
 	std::string extension;
 
-	if (argc >= 3)
-	{
-		// OUTFILE
-		outfile = argv[2];
-		type = GetFiletype(infile, extension);
+	for (int i = 0; i < argc; ++i) {
+		if (category == FileCategory_Invalid) {
+			category = GetFilecategory(argv[i]);
+			if (category == FileCategory_Invalid) {
+				continue;
```

Comment:
Since `lcf2xml` is mostly a developer tool, I would suggest adding a debug message here.